extensions/toggl-track/src/helpers/preferences.ts

```diff
@@ -1,8 +1,12 @@
 import { getPreferenceValues } from "@raycast/api";
 
 const preferences = getPreferenceValues<Preferences>();
-export const togglApiToken = preferences.togglApiToken;
+export const { togglApiToken, showClientsInForm, showProjectsInForm, showTasksInForm, showTagsInForm } = preferences;
 
 interface Preferences {
   togglApiToken: string;
+  showClientsInForm: boolean;
+  showProjectsInForm: boolean;
+  showTasksInForm: boolean;
+  showTagsInForm: boolean;
 }
```

Comment:
```suggestion
```
You don't need to manually set preference types as this is autogenerated in `raycast-env.d.ts`